CONTRIBUTING.md

```diff
@@ -20,22 +20,25 @@ There are two files that might be helpful to investigate before contributing: [_
 
```

Comment:
The scss code block above this line here is indented by one space and messes up the syntax highlighting for the rest of the file. Not your fault but it is a quick fix to just remove the space before the start and end ``` (and I guess before the .some-guy class).